src/main/java/uk/gov/hmcts/reform/pip/subscription/management/services/SubscriptionService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.pip.subscription.management.services;
```

Comment:
on this folder naming, `service` instead of `services` - we've just been keeping it singular and referring to it as "the service folder"